backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -51,45 +59,35 @@
 import org.hl7.fhir.r4.model.Enumerations.AdministrativeGender;
 import org.hl7.fhir.r4.model.Extension;
 import org.hl7.fhir.r4.model.HumanName;
-import org.hl7.fhir.r4.model.Identifier;
-import org.hl7.fhir.r4.model.Identifier.IdentifierUse;
+import org.hl7.fhir.r4.model.MessageHeader;
 import org.hl7.fhir.r4.model.Observation;
 import org.hl7.fhir.r4.model.Observation.ObservationStatus;
+import org.hl7.fhir.r4.model.Organization;
+import org.hl7.fhir.r4.model.Patient;
+import org.hl7.fhir.r4.model.Practitioner;
+import org.hl7.fhir.r4.model.PractitionerRole;
+import org.hl7.fhir.r4.model.Reference;
+import org.hl7.fhir.r4.model.Resource;
+import org.hl7.fhir.r4.model.ResourceType;
 import org.hl7.fhir.r4.model.ServiceRequest;
 import org.hl7.fhir.r4.model.ServiceRequest.ServiceRequestIntent;
 import org.hl7.fhir.r4.model.ServiceRequest.ServiceRequestStatus;
 import org.hl7.fhir.r4.model.Specimen;
 import org.hl7.fhir.r4.model.StringType;
+import org.springframework.data.util.Pair;
 
 @Slf4j
 public class FhirConverter {
   private FhirConverter() {
     throw new IllegalStateException("Utility class");
   }
 
-  public static Identifier convertToIdentifier(UUID id) {
-    if (id != null) {
-      return convertToIdentifier(id.toString());
-    }
-    return null;
-  }
-
-  public static Identifier convertToIdentifier(String id) {
-    if (id != null) {
-      return new Identifier().setValue(id).setUse(IdentifierUse.USUAL);
-    }
-    return null;
-  }
-
-  public static HumanName convertToHumanName(PersonName personName) {
-    if (personName != null) {
-      return convertToHumanName(
-          personName.getFirstName(),
-          personName.getMiddleName(),
-          personName.getLastName(),
-          personName.getSuffix());
-    }
-    return null;
+  public static HumanName convertToHumanName(@NotNull PersonName personName) {
```

Comment:
nit - most of the other methods follow the pattern of `convert<thing>To<otherThing>` and this one is missing the first <thing>